sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2048,6 +2048,16 @@ object SQLConf {
     .checkValue(threshold => threshold > 0, "The threshold must be a positive integer.")
     .createWithDefault(1024)
 
+  val INPUT_CODEGEN_RDD_INTERRUPT_ON_CANCEL =
+    buildConf("spark.sql.codegen.inputCodegenRDD.interruptOnCancel")
```

Comment:
Do we have a plan to introduce more configurations under `spark.sql.codegen.inputCodegenRDD.*` namespace? If not, maybe, we had better avoid introducing this new namespace, `spark.sql.codegen.inputCodegenRDD.*`.